get-started/installation-procedure.md

```diff
@@ -56,6 +56,32 @@ On most **Linux** distributions before the installation can be started, the user
     ![KDE setting executible](../images/installation/exec-kde.jpg)
 
 
+## Create an Account
+
+Fiddler Everywhere account enables you to save and share sessions with collaborators and additionally provides access to the Pro features (if Pro version is purchased). Signing in is mandatory for both Free and Pro versions. You can sign up by [creating an unique Fiddler account](#create-new-fiddler-account) or by [using existing Google account](#use-google-account).
+
+### Create New Fiddler Account
+
+1. Launch Fiddler Everywhere application. The initial screen shows the **Sign In & Create Account** options. Click on **Create Account**.
```

Comment:
Launch Fiddler Everywhere application -> "**the** Fiddler Everywhere application" or just "Fiddler Everywhere". Also, consider using "client" instead of "application" for consistency throughout the docs.